model_server/app/main.py

```diff
@@ -15,12 +16,42 @@
 )
 from unittest.mock import patch
 
+from opentelemetry import trace
```

Comment:
Please remove tracing from model_server internal calls. Not necessary

Reply:
its not used - see this line https://github.com/katanemo/arch/pull/229/files#diff-a883497ecfcfe46bc8a13b33b682d89dc175ff777398e7f8fce68cfc990087f1R46